README.md

```diff
@@ -34,3 +34,8 @@ Make company global travel and accommodation easy and convenient for the strong
     npm run start
   ```
 
+ Coveralls for code coverage
+- Clone the repo
+- create a file .coveralls.yml
+- Untrack the `repo_token` from .gitignore file
```

Comment:
@davidfrank96  `Untrack the `repo_token` from .gitignore file`. I don't understand what this means. You can replace it with how the user can get the `repo_token` and how to put it in the `.coveralls.yml` file